dev_guide/downward_api.adoc

```diff
@@ -293,8 +292,8 @@ field to describe the desired resources that correspond to the
 `*spec.resources*` field:
 +
 ====
+[source,yaml]
 ----
-....
```

Comment:
this is here for a reason, it indicates this is not a fully complete pod definition.